src/modules/pos/validator/store/actions/staking.test.js

```diff
@@ -63,7 +65,7 @@ describe('actions: staking', () => {
           sequence: {
             nonce: '1',
           },
-          votes: [{ delegateAddress: '123L', amount: 1e9 }],
+          stakes: [{ validatorAddress: '123L', amount: 1e9 }],
```

Comment:
stakes will never have `validatorAddress`? http://165.227.246.146:9901/api/v3/pos/stakes?address=lsk37av46nmhr979948rvp3zx7n39xpetoqnqrxny

Reply:
ok i think this requires refactoring, so ill create a separate issue.